doc/release-process.md

```diff
@@ -1,18 +1,26 @@
 Release Process
 ====================
 
+Just after branching off a new release:
+
+* In `configure.ac` and `build_msvc/bitcoin_config.h` on _the master branch_:
+  - update `CLIENT_VERSION_MINOR` version
+* In `configure.ac` and `build_msvc/bitcoin_config.h` on _a new release branch_:
+  - update `CLIENT_VERSION_MINOR` version
+  - set `CLIENT_VERSION_REVISION` to `0`
+  - set `CLIENT_VERSION_IS_RELEASE` to `true`
```

Comment:
Could link to an example commit 742f7dd972fca3dd4a33cfff90bf901b71a687e7?

Could move this down to section *Before a major release*?